Branch-SDK/Branch-SDK/BNCCommerceEvent.m

```diff
@@ -229,6 +255,203 @@
 BNCCurrency BNCCurrencyZAR = @"ZAR";
 BNCCurrency BNCCurrencyZMW = @"ZMW";
 
+NSArray<BNCCurrency>* BNCCurrencyAllCurrencies(void) {
```

Comment:
Should we change this to an enumeration instead?

Reply:
This is the Swift compatible way to do enums.

Reply:
Got it